a11y-input/test/ally-input.test.js

```diff
@@ -78,3 +78,33 @@ describe('a11y input', () => {
     expect(el.value).to.equal('foo');
   });
 });
+
+/**
+ *
+ * Adding test stub to increase `Statements` `code coverage`:
+ *
+ * =============================== Coverage summary ===============================
+ * Statements   : 78.57% ( 22/28 )
+ * Branches     : 37.5% ( 3/8 )
+ * Functions    : 87.5% ( 7/8 )
+ * Lines        : 78.57% ( 22/28 )
+ * ================================================================================
+ *
+ * 24 03 2020 11:31:30.693:ERROR [reporter.coverage-istanbul]:
+ *  Coverage for statements (78.57%) does not meet global threshold (90%)
+ * 24 03 2020 11:31:30.693:ERROR [reporter.coverage-istanbul]:
+ *  Coverage for lines (78.57%) does not meet global threshold (90%)
+ * 24 03 2020 11:31:30.693:ERROR [reporter.coverage-istanbul]:
+ *  Coverage for branches (37.5%) does not meet global threshold (90%)
+ * 24 03 2020 11:31:30.693:ERROR [reporter.coverage-istanbul]:
+ *  Coverage for functions (87.5%) does not meet global threshold (90%)
+ */
+describe('ally input', () => {
+  it('logs "We like cats too 🐱" if the value is "cat"', async () => {
+    const el = /** @type {A11yInput} */ (await fixture(html`
+      <ally-input .value=${'cat'}></ally-input>
```

Comment:
### Looks like I've been carrying forward a bug...

### Source should be `<a11y-input>` with 1s not `<ally-input>` the letter `l`